sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/prerequisites/manifest.json

```diff
@@ -0,0 +1,9 @@
+[{
```

Comment:
we no longer need this manifest, and even if we do ( I highly doubt it) these values are specific to ADT